wom/src/main/scala/wdl4s/wdl/Declaration.scala

```diff
@@ -6,6 +6,8 @@ import wdl4s.parser.WdlParser.{Ast, AstNode}
 import wdl4s.wdl.AstTools.EnhancedAstNode
 import wdl4s.wdl.types.{WdlArrayType, WdlOptionalType, WdlType}
 import wdl4s.wom.graph._
+// Lets me imply that something should be a Some(), but still get the warm fuzzy feeling of wrapping in Option.apply()
+import scala.{Option => SafeSome}
```

Comment:
Please no

Reply:
For a "safe" option, how about `Option(foo).orElse(throw new MyException())`. That way a customizable exception is thrown (with a stack trace!) iff `foo` is somehow `null`.

Reply:
In my mind I'd still prefer people just using `Some` than us having some sort of special hijinks to make both sides happy. Perhaps this is @cjllanwarne trying to keep proposing increasingly horrible suggestions until we view `Some` as not so bad :)

Reply:
I'll just use `Option` this time. 

FWIW, the reason I don't `Option` it is the same reason you wouldn't replace `None` with `Option(null)` ("but what if `null` ends up having a real value someday?!"), so the `.orElse` way offends me even more I'm afraid!